### PR TITLE
Organization blog posts

### PR DESCRIPTION
**Pull request by hung3a8**
Organization admins with "edit_organization_post" permission can create organization posts that only display inside organization views.
Preview:
image
image
image

### CODE REVIEW

File: judge/migrations/0154_blog_post_org.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='blogpost',
+            options={'permissions': (('edit_all_post', 'Edit all posts'), ('edit_organization_post', 'Edit organization posts')), 'verbose_name': 'blog post', 'verbose_name_plural': 'blog posts'},
+        ),
+        migrations.AddField(
+            model_name='blogpost',
+            name='global_post',
+            field=models.BooleanField(default=False, help_text='Display this blog post at the homepage.', verbose_name='global post'),
```
Comment (leduythuccs) [inline judge/migrations/0154_blog_post_org.py:21]: The `default=False` make all old blog became hidden to users, so try to run a python script here.

**Review by leduythuccs — CHANGES_REQUESTED** [review]

File: templates/blog/personal-list.html [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+        .own-open-tickets .title a, .open-tickets .title a {
+            display: block;
+        }
+
+        .own-open-tickets .object, .open-tickets .object {
+            margin-left: 1em;
+            font-style: italic;
+        }
+
+        .open-tickets .author {
```
Comment (leduythuccs) [inline templates/blog/personal-list.html:41]: remove all unused css

File: templates/blog/personal-list.html [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+        table.bottom td.right {
+            text-align: right;
+            border-top: none !important;
+        }
+    </style>
+{% endblock %}
+
+{% block js_media %}
+    <script type="text/javascript">
+        $(document).ready(function () {
```
Comment (leduythuccs) [inline templates/blog/personal-list.html:60]: I think this js is only used on the homepage, so, please remove it.

**Review by leduythuccs — CHANGES_REQUESTED** [review]

**Review by hieplpvip — APPROVED** [review]
Looks good to me

**Review by leduythuccs — APPROVED** [review]
LGTM

**Pull request merged**